SYCL/ESIMD/api/functional/ctors/ctor_copy.cpp

```diff
@@ -33,11 +33,11 @@ struct initializer {
   static std::string get_description() { return "initializer"; }
 
   template <typename DataT, int NumElems>
-  static simd<DataT, NumElems> call_simd_ctor(const DataT *ref_data) {
+  static void call_simd_ctor(const DataT *ref_data, DataT *out) {
```

Comment:
```suggestion
  static void call_simd_ctor(const DataT *const ref_data, DataT * const out) {
```
What would you say to make the constness of the pointer itself explicit? For each test case.

Reply:
I've applied this comment in this commit: https://github.com/intel/llvm-test-suite/pull/614/commits/e87ad2ec4b8cff0070719ae8992d50d244c2a019